doc/user-manual/team.rst

```diff
@@ -3,42 +3,75 @@
+
+Agda 2 is currently actively developed mainly by (in alphabetical order):
 
-* Ulf Norell
 * Andreas Abel
+* Guillaume Allais
+* Liang-Ting Chen
+* Jesper Cockx
 * Nils Anders Danielsson
-* Makoto Takeyama
-* Catarina Coquand
+* Ulf Norell
+* Víctor López Juan
+* Andrés Sicard-Ramírez
+* Andrea Vezzosi
+* Tesla Ice Zhang
 
-Contributors (alphabetically sorted):
+Agda 2 has received major contributions by the following developers, amongst others.
+Some contributors have pioneered a feature which shall be mentioned here.
+But many have worked on these features for improvements and maintenance.
 
-* Stevan Andjelkovic
-* Marcin Benke
-* Jean-Philippe Bernardy
+* Andreas Abel: *termination checker, sized types, irrelevance, copatterns*
+* Guillaume Allais: *warnings, pattern guards, standard library 1.0*
+* Stevan Andjelkovic: *LaTeX backend*
+* Miëtek Bak: *Agda logo*
+* Marcin Benke: *original "Alonzo" compiler to Haskell*
+* Jean-Philippe Bernardy: ``syntax`` *declarations*
+* Guillaume Brunerie
 * James Chapman
-* Jesper Cockx
-* Dominique Devriese
-* Péter Diviánszky
-* Fredrik Nordvall Forsberg
-* Olle Fredriksson
-* Daniel Gustafsson
-* Philipp Hausmann
+* Liang-Ting Chen: *github workflows*
+* Jesper Cockx: *rewriting, unification* ``--without-K``
+* Catarina Coquand: *Agda 1*
+* Matthew Daggitt: *standard library 1.0*
+* Nils Anders Danielsson: *efficient positivity checker, HTML backend, highlighting, standard library*
+* Dominique Devriese: ``instance`` *arguments*
+* Péter Diviánszky: *web frontent,* ``variable`` *declarations*
+* Robert Estelle
+* Olle Fredriksson: *Epic compiler backend*
+* Paolo Giarrusso
+* Adam Gundry: *pattern synonyms*
+* Daniel Gustafsson: *Epic compiler backend*
+* Philipp Hausmann: *treeless compiler, UHC compiler backend, testsuite runner, Travis CI*
+* Kuen-Bang Hou "favonia"
 * Patrik Jansson
-* Alan Jeffrey
+* Alan Jeffrey: *JavaScript compiler backend*
 * Wolfram Kahl
+* Wen Kokke
 * John Leo
-* Fredrik Lindblad
-* Francesco Mazzoli
+* Fredrik Lindblad: *Agsy proof search "Auto"*
+* Víctor López Juan: "tog" prototype, markdown frontend, documentation*
+* Ting-Gan Lua
+* Francesco Mazzoli: "tog" prototype*
 * Stefan Monnier
-* Darin Morrison
-* Guilhem Moulin
-* Nicolas Pouillard
+* Guilhem Moulin: *highlighting*
+* Fredrik Nordvall Forsberg: *pattern lambdas, warnings*
+* Ulf Norell: *Agda 2*
+* Nicolas Pouillard: *module record expressions*
 * Benjamin Price
-* Andrés Sicard-Ramírez
-* Andrea Vezzosi
+* Jonathan Prieto: *Agda package manager*
+* Nobuo Yamashita
+* Christian Sattler
+* Andrés Sicard-Ramírez: *Agda releases, Travis CI*
+* Makoto Takeyama: *Agda 1, Emacs mode, "MAlonzo" compiler to Haskell, serialization*
+* Andrea Vezzosi: *Cubical Agda, Agda-flat, Agda-parametric*
+* Noam Zeilberger: *pattern lambdas*
 * Tesla Ice Zhang
-* and many more
+
+The full list of contributors (more than 140)
+is available at https://github.com/agda/agda/graphs/contributors .
```

Comment:
Since the link points to the current contributors, future contributors will have copyright claims to previous versions where they hadn't contributed.

Reply:
Interesting thought, but is this so?
You can only have copyright on code you actually wrote, isn't it?
And authorship can be tracked via git.

Reply:
Probably, you are right. And I doubt it will matter anyway.